ui/userunlock.js

```diff
@@ -97,7 +97,7 @@ function prompt_sign_in() {
         buttons: {
             formSubmit: {
                 text: 'Sign In',
-                btnClass: 'btn-blue',
+                btnClass: 'btn-blue sign-in',
```

Comment:
Why add `sign-in` in the class here? What does it achieve?

Reply:
https://github.com/rotkehlchenio/rotkehlchen/pull/136#discussion_r213509599